tests/fixtures/fixtures_spawner.py

```diff
@@ -757,15 +759,22 @@ def let_next_spawn_fail(self):
     return MockTrainSpawner()
 
 
+@pytest.fixture
+def event_bus(run) -> EventBus:
+    bus = EventBus(run_id=run.id)
+    Logger(run.id, bus)
+    return bus
```

Comment:
This is the same fixture as in fixtures/fixtures_logger.py. Why do you need it again?

Reply:
That seems good.